torch/distributed/tensor/parallel/style.py

```diff
@@ -482,8 +482,8 @@ class RowwiseParallel(ParallelStyle):
 
     def __init__(
         self,
-        _prepare_input=None,
-        _prepare_output=None,
```

Comment:
But this somehow revert the `input_layouts` and `output_layouts`? So does this mean when users want to use `input_layouts` and `output_layouts`, they have to set `_prepare_input` to `None`?

Reply:
This is just a quick fix, I'll do a big rewrite on top to make sure user behavior won't change

Reply:
I see.

Reply:
OK I gave up on the big rewrite and found a quicker solution right now, however it does not fundamentally fix the torch.compile problem in the new style `input_layouts` and `output_layouts` path (i.e. to configure sequence parallel torch.compile still fail). We should do a full rewrite soon to make the TP code cleaner and tracing friendly

Reply:
this "hack" preserves the intended behavior, but I dislike it..